dynawo/sources/Modeler/DataInterface/DYNComponentInterface.cpp

```diff
@@ -111,6 +124,30 @@ ComponentInterface::getType() const {
   return type_;
 }
 
+const std::string&
+ComponentInterface::getTypeAsString() const {
+  static const std::string stringTypes[] = {
+      "UNKNOWN",
+      "BUS",
+      "CALCULATED_BUS",
```

Comment:
CALCULATED_BUS should be represented by opened switches

Reply:
I don't understand the remark !
This type already exists, so it has a string representation too.

Reply:
The calculated_bus is a fictive component generated to represent a set of switches and bus bar sections, thus it has no meaning outside of Dynawo